### PR TITLE
Add Docker recipe

### PR DESCRIPTION
**Pull request by texnixe**
I have created a monster...

If anyone with a Windows device has a chance to test the last example, that would be huge.

### CODE REVIEW

**Review by bastianallgeier — APPROVED** [review]
Wow, this is incredible! I've never really dared diving deeper into Docker containers and this intro is so useful. <3

File: content/docs/2_cookbook/9_setup/0_kirby-meets-docker/cookbook-recipe.txt [hunk 1]
```diff
@@ -0,0 +1,949 @@
...
+      - ./starterkit:/var/www/html/
+    env_file:
+      - ./id.env
+```
+
+### `docker-compose.yml` explained
+
+A `docker-compose.yml` file always starts with the Docker Compose version number. The current minor version at the time of writing this recipe is 3.8.
+
+After that, the next top level keyword is `services`, which is followed by an indented list of named services. The names of these services are arbitrary, but of course it makes sense to give them names that indicate their purpose. In the file above, we currently have only one service called `web server`.
```
Comment (afbora) [inline content/docs/2_cookbook/9_setup/0_kirby-meets-docker/cookbook-recipe.txt]:
```suggestion
After that, the next top level keyword is `services`, which is followed by an indented list of named services. The names of these services are arbitrary, but of course it makes sense to give them names that indicate their purpose. In the file above, we currently have only one service called `webserver`.
```

**Review by afbora — APPROVED** [review]

**Comment by bastianallgeier** [comment]
Can we merge it? @texnixe should I already promote it today through Discord and Twitter or wait until the newsletter's out tomorrow?

**Comment by texnixe** [comment]
@bastianallgeier: Let's see if @lukasbestle finds the time to proof-read it this morning, if not, then we just merge it and fix potential typos later.

**Pull request merged**